### DISCUSSION TITLE
cmd/01.cmd

### DISCUSSION THREAD

**Discussion by giscus** [RuanZhongNan/RPGMV-dev-notes#145] [Announcements]
# cmd/01.cmd

常用的 cmd 命令 set-ExecutionPolicy RemoteSigned https://blog.csdn.net/qq_31319235/article/details/119725835 在 powershell.exe 内设置命令行的权限。即可在 VScode 内使用各类命令。如 vue cli 的命令。 tree 树状图输出命令...

https://ruan-cat-notes.ruan-cat.com/cmd/01.cmd.html

**Comment by ZHEGUO6** [comment]
而各色各色
